### PR TITLE
Fixing arrows office mng

### PR DESCRIPTION
**Pull request by ShirAmit96**

### CODE REVIEW

File: frontend/src/Pages/OfficeManagerScreen.css [hunk 1]
```diff
@@ -177,4 +161,4 @@
   line-height: 150%;
   color: #00406B;
 
-}
+}
```
Comment (nitzansavr) [inline frontend/src/Pages/OfficeManagerScreen.css:164]: add a new line

File: frontend/src/Pages/OfficeManagerScreen.js [hunk 2]
```diff
@@ -3,71 +3,112 @@ import "../components/calendarElement.js";
...
 import leftArrow from "../Icons/leftArrow.svg";
 import rightArrow from "../Icons/rightArrow.svg";
+import { useDispatch, useStore } from "react-redux";
+import axios from "axios";
 import { useSelector } from "react-redux";
-import { selectDate } from "../redux/dateSlice.js";
-
-
+import { selectDate, selectedDate } from "../redux/dateSlice.js";
+const postReq = async (dateText) => {
```
Comment (nitzansavr) [inline frontend/src/Pages/OfficeManagerScreen.js:13]:
```suggestion

const postReq = async (dateText) => {
```

File: frontend/src/Pages/OfficeManagerScreen.js [hunk 3]
```diff
@@ -3,71 +3,112 @@ import "../components/calendarElement.js";
...
+  const array = await axios.post("http://localhost:5000/dates/getPerDate", {
```
Comment (nitzansavr) [inline frontend/src/Pages/OfficeManagerScreen.js:14]: please extract the URL to a constant

File: frontend/src/Pages/OfficeManagerScreen.js [hunk 4]
```diff
@@ -3,71 +3,112 @@ import "../components/calendarElement.js";
...
+    date: dateText,
+  });
+  return array;
+};
 function OfficeManagerScreen() {
   const [date, setDate] = useState(new Date());
+  const WEEKDAY = [
```
Comment (nitzansavr) [inline frontend/src/Pages/OfficeManagerScreen.js:21]: these constants (WEEKDAY and MONTH) should be in the top of the file

File: frontend/src/Pages/OfficeManagerScreen.js [hunk 5]
```diff
@@ -3,71 +3,112 @@ import "../components/calendarElement.js";
...
-              newDate.setDate(day + 1)
-              setDate(newDate)
-            }
-          }} className="nextArrow" />
+          <img
+            src={rightArrow}
+            onClick={() => {
+              const day = date.getDate();
+              const newDate = new Date(date);
+              if (day + 1 === 5) {
```
Comment (nitzansavr) [inline frontend/src/Pages/OfficeManagerScreen.js:97]: please add a descriptive comment about why you need this special logic

File: frontend/src/Pages/OfficeManagerScreen.js [hunk 6]
```diff
@@ -3,71 +3,112 @@ import "../components/calendarElement.js";
...
+                newDate.setDate(day + 2);
+                setDate(newDate);
+              } else {
+                newDate.setDate(day + 1);
+                setDate(newDate);
+              }
+              //  newDate.setDate(day+1)
+              //  setDate(newDate)
```
Comment (nitzansavr) [inline frontend/src/Pages/OfficeManagerScreen.js:105]: I think you can remove this commented out code

File: frontend/src/components/calendarElement.js [hunk 7]
```diff
@@ -1,51 +1,31 @@
...
-          sum: transferInfo.data.length,
-        })
-      );
-    })
-  }, [date]);
+import React, { useState } from "react";
+import Calendar from "react-calendar";
+import "./calendarElement.css";
+function MyCalendar(props) {
+  //const [date, setDate] = useState(new Date());
```
Comment (nitzansavr) [inline frontend/src/components/calendarElement.js:5]: I believe you can remove this commented out code

File: frontend/src/components/calendarElement.js [hunk 8]
```diff
@@ -1,51 +1,31 @@
...
-          tileDisabled={({activeStartDate, date, view }) => {
-            const day = date.getDay()
-            return (day === 6 || day === 5);
-          }
-          }
+            return date.toLocaleDateString("en-us", { weekday: "short" });
+          }}
+          tileDisabled={({ activeStartDate, date, view }) => {
+            const day = date.getDay();
+            return day === 6 || day === 5;
```
Comment (nitzansavr) [inline frontend/src/components/calendarElement.js:27]: add a descriptive comment. For example, `# disable tile on weekend`

File: frontend/src/components/calendarForReg.js [hunk 9]
```diff
@@ -8,44 +8,50 @@ import axios from "axios";
...
-          }
+            return date.toLocaleDateString("en-us", { weekday: "short" });
+          }}
           tileDisabled={({ activeStartDate, date, view }) => {
-            const day = date.getDay()
-            return (day === 6 || day === 5);
-          }
-          }
+            const day = date.getDay();
+            return day === 6 || day === 5;
```
Comment (nitzansavr) [inline frontend/src/components/calendarForReg.js:52]: It is a duplicated code (same as lines 26-27 in `frontend/src/components/calendarElement.js`). Is it possible to extract it to a single util function?

File: frontend/src/Pages/OfficeManagerScreen.js (hunk 2 above)
Comment (ShirAmit96) [reply frontend/src/Pages/OfficeManagerScreen.js:13]: I can't see the difference

Comment (nitzansavr) [reply frontend/src/Pages/OfficeManagerScreen.js:36]: a new line under the imports

File: frontend/src/components/calendarForReg.js (hunk 9 above)
Comment (Lielmv) [reply frontend/src/components/calendarForReg.js:52]: will be fixed in another branch

**Review by katz-tom — APPROVED** [review]

**Pull request merged**